### DISCUSSION TITLE
SNlM0e value not found

### DISCUSSION THREAD

**Discussion by gowtham-r-probe** [dsdanielpark/Bard-API#219] [Q&A]
I am still getting this error SNlM0e value not found. Double-check __Secure-1PSID value or pass it as token='xxxxx'.

Any suggestions.

**Comment by gowthamr79** [comment]
Even I am running into same issue, may I know if this current setup still running on others system. I am actually about to integrate this into my productions.

**Comment by dsdanielpark** [comment]
The issue at hand is that Google is not providing a normal response depending on the region, account, etc. As a result, we can no longer offer support at the package level. Therefore, we recommend trying various methods to resolve it.

I hope you are finding well in this document.

https://github.com/dsdanielpark/Bard-API/blob/main/documents/README_FAQ.md
